libobs/CMakeLists.txt

```diff
@@ -170,10 +171,26 @@ elseif(APPLE)
 elseif(UNIX)
 	set(libobs_PLATFORM_SOURCES
 		obs-nix.c
+		obs-nix-x11.c
 		util/threading-posix.c
 		util/pipe-posix.c
 		util/platform-nix.c)
 
+	if(ENABLE_WAYLAND)
+		find_package(Wayland REQUIRED)
```

Comment:
use the proper components. Only client is required here.

Reply:
As discussed on Discord, this is done, but the price is the lack of sanity in `FindWayland.cmake`